app/src/main/java/com/github/siela1915/bootcamp/AutocompleteApi/IngredientAutocomplete.java

```diff
@@ -14,43 +14,44 @@
 public class IngredientAutocomplete {
     private final int numberOfIngredients = 5;
     private final String apiKey = "44a82829c64d4202a18b887e47e76bdd";
```

Comment:
Api key in code on a public repository is a bad idea, should try to move it to an environment variable or something